padre_meddea/calibration/calibration.py

```diff
@@ -320,3 +320,162 @@ def read_calibration_file(calib_filename: Path):
     # if can't read the file
 
     return None
+
+
+def concatenate_daily_fits(
+    files_to_combine: list[Path], existing_file: Path = None, outfile: Path = None
+) -> Path:
+    """
```

Comment:
please add a Note that states that this assumes that there is no data stored in the PrimaryHDU.